templates/page.tpl.php

```diff
@@ -105,10 +105,15 @@
 
 <footer role="contentinfo">
   <div class="wrapper">
-    <section class="footer-top">
-      <?php print render($page['footer_top']); ?>
-    </section>
+    <?php if($page['footer_top']): ?>
+      <section class="footer-top">
+        <?php print render($page['footer_top']); ?>
+      </section>
+    <?php endif; ?>
     <section class="footer-bottom">
```

Comment:
You should lose a biscuit for including a change not released to your specific issue :)